mdr/mdr.py

```diff
@@ -57,6 +60,22 @@ def fit(self, features, classes):
         None
 
         """
+        n = (np.unique(classes)).size # general case 
+        for i in range(features.shape[0]):
+            feature_instance = tuple(map(tuple, features[i])) 
+            if feature_instance not in dict: 
+                feature_map[feature_instance] = np.zeros((n,), dtype=np.int) #initialize count for a new set of feature values 
+            #assuming there are only 2 unique label values - might have to change to generalize
+            if classes[i] = 0:
+                feature_map[feature_instance][0] += 1
+            else:
+                feature_map[feature_instance][1] += 1
```

Comment:
This can be replaced with one line: `feature_map[feature_instance][classes[i]] += 1`